laws/shared/src/main/scala/algebra/laws/RunSerializationLaws.scala

```diff
@@ -0,0 +1,9 @@
+package algebra.laws
+
+sealed trait SerializationCheck
```

Comment:
Why do we need the type parameter? I had goofed and set it to `Skip` above during some cycling when running the tests, just to make sure that this still took effect. Switching to `Run` by default works great.